psutil/_psutil_windows.c

```diff
@@ -1278,31 +1278,31 @@ psutil_proc_open_files(PyObject *self, PyObject *args) {
 
 
 /*
- Accept a filename's drive in native  format like "\Device\HarddiskVolume1\"
+ Accept a filename's drive in native format like "\Device\HarddiskVolume1\"
  and return the corresponding drive letter (e.g. "C:\\").
  If no match is found return an empty string.
 */
 static PyObject *
 psutil_win32_QueryDosDevice(PyObject *self, PyObject *args) {
-    LPCTSTR   lpDevicePath;
+    LPCWSTR   lpDevicePath;
     TCHAR d = TEXT('A');
     TCHAR     szBuff[5];
 
-    if (!PyArg_ParseTuple(args, "s", &lpDevicePath))
+    if (!PyArg_ParseTuple(args, "u", &lpDevicePath))
         return NULL;
 
     while (d <= TEXT('Z')) {
         TCHAR szDeviceName[3] = {d, TEXT(':'), TEXT('\0')};
-        TCHAR szTarget[512] = {0};
-        if (QueryDosDevice(szDeviceName, szTarget, 511) != 0) {
-            if (_tcscmp(lpDevicePath, szTarget) == 0) {
+        WCHAR szTarget[512] = {0};
+        if (QueryDosDeviceW(szDeviceName, szTarget, 511) != 0) {
+            if (wcscmp(lpDevicePath, szTarget) == 0) {
                 _stprintf_s(szBuff, _countof(szBuff), TEXT("%c:"), d);
-                return Py_BuildValue("s", szBuff);
+                return Py_BuildValue("u", szBuff);
```

Comment:
Does this mean the return type on Python 2 is always gonna be unicode (instead of str)?